staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/forklift/v1beta1/types.go

```diff
@@ -0,0 +1,85 @@
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OvirtVolumePopulatorKind is the type of the CR used to populator a volume from an oVirt disk
+var OvirtVolumePopulatorKind = "OvirtVolumePopulator"
+
+// OvirtVolumePopulator is the CR used to populator a volume from an oVirt disk
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:openapi-gen=true
+// +kubebuilder:resource:shortName={ovvp,ovvps}
+type OvirtVolumePopulator struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec OvirtVolumePopulatorSpec `json:"spec"`
+	// +optional
+	Status OvirtVolumePopulatorStatus `json:"status"`
```

Comment:
For all fields that are optional should probably add "omitempty" to json tags as well.  I don't think the json generation code looks at "// + optional" think that's just code gen tools

Reply:
added